.storybook/config.js

```diff
@@ -1,29 +1,29 @@
-import { configure, setAddon, addDecorator } from "@storybook/react";
-import infoAddon from "@storybook/addon-info";
-import React from "react";
-import { Provider } from "../src/index";
-import theme from "../src/theme";
-import { injectGlobal } from "styled-components";
+import { configure, setAddon, addDecorator } from '@storybook/react'
+import infoAddon from '@storybook/addon-info'
+import * as React from 'react'
```

Comment:
Can't we import .default with type script?

Reply:
I don't mind the syntax either way really, just find it weird that you can't do the same as you can do in es6? I see there is a `--allowSyntheticDefaultImports` would that allow us to do `import React from 'react'`?

Reply:
Babel does some magic where it uses an interOp to normalize these kinds of imports. The typescript compiler doesn't do this AFAIK